spec/requests/api/v2/providers/courses_spec.rb

```diff
@@ -154,6 +154,75 @@
     end
   end
 
+  describe 'POST publish' do
+    let(:api_status)   { 200 }
+    let(:api_response) { '{ "result": true }' }
+    let!(:stubbed_manage_courses_api_request) do
```

Comment:
Why let! Instead of let ?

Reply:
Good catch. Also, I'd say, `let` should be used for memoized values, not for setup. Especially if the setup is non-optional like here.